pkg/component/observability/monitoring/prometheus/component.go

```diff
@@ -76,6 +76,8 @@ type Values struct {
 	RetentionSize monitoringv1.ByteSize
 	// RuntimeVersion is the Kubernetes version of the runtime cluster.
 	RuntimeVersion *semver.Version
+	// ScrapeTimeout is the timeout duration when scraping targets.
+	ScrapeTimeout monitoringv1.Duration
```

Comment:
TBH, I copied this explanation from @istvanballok's previous work on this. So far we don't have validation for values of components. I can add it if you insist, but it would be inconsistent.